flink-sql-runner/src/test/java/com/github/streamshub/flink/SqlRunnerTest.java

```diff
@@ -60,38 +64,40 @@ void testParseStatementsWithSecrets () {
-                        "  'scan.startup.mode' = 'earliest-offset'," +
-                        "  'format' = 'csv'); " +
-                "CREATE TABLE print_table ( message STRING ) WITH ('connector' = 'print'); " +
-                "CREATE TABLE print_table2 ( message STRING ) WITH ('connector' = 'print'); " +
-                "EXECUTE STATEMENT SET BEGIN INSERT INTO print_table SELECT * FROM KafkaTable; " +
-                "INSERT INTO print_table2 SELECT * FROM print_table; END; ";
+                "CREATE TABLE KafkaTable (\nmessage STRING\n) WITH (\n'connector' = 'kafka'\n" +
+                        "\t'topic' = 'user_behavior',\n" +
+                        "\t'properties.bootstrap.servers' = 'localhost:9092',\n" +
+                        "\t'properties.group.id' = 'testGroup',\n" +
+                        "\t'scan.startup.mode' = 'earliest-offset',\n" +
+                        "\t'format' = 'csv'\n);\n" +
+                "CREATE TABLE print_table ( message STRING ) WITH ('connector' = 'print');\n" +
+                "CREATE TABLE print_table2 ( message STRING ) WITH ('connector' = 'print');\n" +
+                "EXECUTE STATEMENT SET\nBEGIN\n\tINSERT INTO print_table SELECT * FROM KafkaTable;\n" +
+                "\tINSERT INTO print_table2 SELECT * FROM print_table;\nEND;\n" +
+                "SELECT *\nFROM print_table2;";
 
         List<String> expected = List.of(
-                "CREATE TABLE KafkaTable ( message STRING ) WITH ('connector' = 'kafka'" +
-                        "  'topic' = 'user_behavior'," +
-                        "  'properties.bootstrap.servers' = 'localhost:9092'," +
-                        "  'properties.group.id' = 'testGroup'," +
-                        "  'scan.startup.mode' = 'earliest-offset'," +
-                        "  'format' = 'csv');",
+                "CREATE TABLE KafkaTable ( message STRING ) WITH ( 'connector' = 'kafka' " +
+                        "'topic' = 'user_behavior', " +
+                        "'properties.bootstrap.servers' = 'localhost:9092', " +
+                        "'properties.group.id' = 'testGroup', " +
+                        "'scan.startup.mode' = 'earliest-offset', " +
+                        "'format' = 'csv' );",
                 "CREATE TABLE print_table ( message STRING ) WITH ('connector' = 'print');",
                 "CREATE TABLE print_table2 ( message STRING ) WITH ('connector' = 'print');",
                 "EXECUTE STATEMENT SET BEGIN INSERT INTO print_table SELECT * FROM KafkaTable; " +
-                "INSERT INTO print_table2 SELECT * FROM print_table; END;");
+                "INSERT INTO print_table2 SELECT * FROM print_table; END;",
+                "SELECT * FROM print_table2;");
 
-        List<String> actual = SqlRunner.parseStatements(statementSet);
+        List<String> actual = SqlRunner.parseStatementArgs(new String[]{statementSet});
         assertEquals(expected, actual);
```

Comment:
The PR description says the handling of `SET` statements has changed so they no longer sort to the end but this test hasn't change the verification of order. So I'm confused 😕 

I think the assertion is dependent on the equals implementation of the returned list. So maybe we should use [AssertJ#containsExactly](http://joel-costigliola.github.io/assertj/core-8/api/org/assertj/core/api/ListAssert.html#containsExactly-ELEMENT...-)

Reply:
The order of "statement sets" (not `SET` commands) in the statement list is now preserved, where as previously we moved the (assumed to be only 1) statement set to the end. 

I added a statement _after_ the statement set to make sure the order wasn't changed.